Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM java:8
+
+RUN wget http://mirror.nbtelecom.com.br/apache/jmeter/binaries/apache-jmeter-5.0.tgz
```

Comment:
we should add https and SHA256 shasum verification

Reply:
Done, there is not mirror with valid ssl certificate, but as I check both the checksum & certificate that not necessary.